### DISCUSSION TITLE
Stupid Question

### DISCUSSION THREAD

**Discussion by S0ULL355** [Kounex/obs_station#32] [Q&A]
What browser does OBS Blade use on Android? I'm trying to get around the currently infamous "something went wrong" and I am signed in on all 3 of my browsers. Also, the "something went wrong" is just a Twitch issue and I don't know why.

**Comment by Kounex** [comment]
The package I'm using is the official implementation from Google for integrating WebViews into Flutter - the documentation states:

_On iOS the WebView widget is backed by a WKWebView; On Android the WebView widget is backed by a WebView._

What the WebView on Android is baked by depends on the Android version. On versions prior to 10 it's most likely Chrome based and on later versions it's supposed to be a designated component (likely Chromium based) - but I can't tell for sure though.

The problem itself is strange. It happens with your browser apps as well? Only on twitch?

**Reply by S0ULL355** [reply]
I've had an issue on Streamlabs Chatbot and I reset my password and it worked. I have a feeling it's a too many places logged into the account thing but now I'm only having the issue in Blade. And yes, only on Twitch and they are aware of the issue but I was seeing if I could get around it somehow.

**Reply by S0ULL355** [reply]
So...for a workaround if possible because I am one person to see the new people that come into my chat and interact with them more, is a way to externally log in via an actual web browser (i.e. Chrome, Firefox) or authorize a connection like actual OBS or Chatbots
